integration_tests/src/main/python/hash_aggregate_test.py

```diff
@@ -624,18 +624,18 @@ def test_decimal128_min_max_group_by(data_gen):
             .agg(f.min('b'), f.max('b')))
 
 @ignore_order(local=True)
-@pytest.mark.parametrize('data_gen', [float_gen, double_gen], ids=idfn)
-def test_float_max_reduction_with_nan(data_gen):
+@pytest.mark.parametrize('data_gen', [SetValuesGen(t, [math.nan, None]) for t in [FloatType(), DoubleType()]], ids=idfn)
```

Comment:
What are we doing for structs that have NaNs in them?  The code shows that this is allowed, but I don't see tests for this.

Reply:
> What are we doing for structs that have NaNs in them?

Sorry, which `structs`?